tests/__mocks__/dplan_api_map_options_admin.json

```diff
@@ -0,0 +1,46 @@
+{
```

Comment:
Where is this used? (Wondering the same for some other files in the mocks directory)

Reply:
I can't tell. I just moved it. Cleaning up would be a next step.

Reply:
I think we should make sure we need the files before adding them (apart from that, I agree that we can clean up things later)